docs/app/pages/components/components-sections/buttons/floating-action-button/floating-action-button.component.html

```diff
@@ -0,0 +1,85 @@
+<ux-floating-action-buttons [ngClass]="direction" [direction]="direction">
+
+    <ux-floating-action-button fab-primary icon="hpe-contract" tooltip="Show Actions" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-add" tooltip="Add New Item" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-analytics" tooltip="Show Analytics" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-app" tooltip="Show Overview" container="body" [placement]="placement"></ux-floating-action-button>
+
+</ux-floating-action-buttons>
+
+<hr>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-panel heading="Customize Example...">
+                <div class="row">
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="top">Top</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="right">Right</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="bottom">Bottom</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="left">Left</ux-radio-button>
+                    </div>
+                </div>
+            </accordion-panel>
+        </accordion>
+    </div>
+</div>
+
+<p>
+    Floating Action Buttons are contained in a <code>ux-floating-action-buttons</code> component. 
+    Each action button should be added as child using the <code>ux-floating-action-button</code> component.
+</p>
+
+<p>
+    The <code>fab-primary</code> attribute should be added to the floating action button that will permanently be visible and will be used
+    to display the additional buttons when clicked.
+</p>
+
+<p>The following input can be used to customize the behavior of the <code>ux-floating-action-buttons</code> component:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="direction" type="string" defaultValue="top">
+        Defines direction items should be displayed. Possible values are <code>top</code>, <code>right</code>, <code>bottom</code> or <code>left</code>.
+    </tr>
+</uxd-api-properties>
+
+<p>The following input can be used to customize the appearance of the <code>ux-floating-action-button</code> component:</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="icon" type="string">
```

Comment:
This should be `required="false"`.

Reply:
Adding this doesn't change the UI, a "Required" message is only shown when the value is true, by default it is false